.github/ISSUE_TEMPLATE/possible-bug.yml

```diff
@@ -0,0 +1,57 @@
+name: Report an issue
+description: Report a bug or an issue that you've encountered.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for reporting an issue to one of our products. Please provide all necessary information to reproduce the issue. Without adequate details, your issue may be closed without investigation.
+
+        Issues can be reported for axe-core, [axe Extension](https://www.deque.com/axe/browser-extensions/), and [axe Linter](https://marketplace.visualstudio.com/items?itemName=deque-systems.vscode-axe-linter). If you're having an issue with one of our npm integrations (e.g. `@axe-core/webdriverjs`), please open an issue in https://github.com/dequelabs/axe-core-npm.
+  - type: dropdown
+    id: product
+    attributes:
+      label: Product
+      description: Which product did you encounter the issue?
+      options:
+        - axe-core
+        - axe Extension
+        - axe Linter
+    validations:
+      required: true
+  - type: checkboxes
+    id: latest-axe
+    attributes:
+      label: Lastest Version
+      options:
+        - label: I have tested the issue with the latest version of the product
+          required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Issue Description
+      description: Please include a description of the issue and a page or code snippet where it can be reproduced.
+      value: |
+        #### Expectation
+        Describe what you expected the product to do.
+
+        #### Actual
+        Describe what the product actually does.
+
+        #### How to Reproduce
+        Provide a code sample or link to a webpage that reproduces the issue. Without this, your issue may be closed without investigation.
+
+        #### Additional context
+        Any thing else we should know about the issue?
```

Comment:
Users are going to `ctrl+a` and delete this (or at least want to). Why not create a separate field for each of these?

Reply:
I had it original as a separate field, but taking a look at past issues opened and not every bug needs every one of the fields (Wilco and I certainly don't need every field). It also felt a bit overwhelming to have 3 additional required fields just to fill in the bug. 

User are able to select a different template and use the blank issue one, but it's not prominent in the UI so I didn't want to discourage anyone from filing a bug.

Reply:
Fair enough 🤷